src/main/resources/mybatis/features.xml

```diff
@@ -61,6 +76,35 @@
           <if test="featureId != null">
               and monitoring_location_identifier = #{featureId,jdbcType=VARCHAR}
           </if>
+          <if test="pointLowLeft != null and pointUpRight != null">
+             and ST_Intersects(ST_SetSRID(ST_MakeBox2D(
+                                         ST_GeomFromText(#{pointLowLeft},4326),
+                                         ST_GeomFromText(#{pointUpRight},4326)), 4326),
+                 ST_setSRID(geom,4326))
+          </if>
+          ORDER BY collection_id, feature_id
+          <if test="limit != null">
+              limit #{limit}
+           </if>
+          <if test="startIndex != null">
+             offset #{startIndex}
+          </if>
+           )
+          ORDER BY collection_id, feature_id
+     </sql>
+
+     <sql id="selectCollectionTitleQuery">
```

Comment:
Why is the collection title logic duplicated (also in the selectFeaturesQuery). Causes confusion and potential double reads of the network table.